ladybug_geometry/polyskel.py

```diff
@@ -0,0 +1,558 @@
+# coding=utf-8
+"""
+Implementation of the straight skeleton algorithm as described by Felkel and Obdržálek in their 1998 conference paper Straight skeleton implementation.
```

Comment:
This line is too long. Our limit is 89 characters.

Reply:
I admit these are highlighted as PEP8 violations in my IDE, just been slow to refactor them into shorter sentences. Will work on it.

Reply:
You can just write them in multiple lines. You don't need to refactor the sentence.